shoov/modules/custom/shoov_migrate/plugins/message.inc

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * @file
+ * Support for message destinations.
+ */
+
+/**
+ * Destination class implementing migration into messages.
+ */
+class MigrateDestinationMessage extends MigrateDestinationEntity {
```

Comment:
maybe this can be avoided -- see [example](https://github.com/openscholar/openscholar/blob/SCHOLAR-3.x/openscholar/modules/os/modules/os_migrate_demo/handlers/og_vocab/og_vocab.inc)